### PR TITLE
#165854187 Refactor response model

### PR DESCRIPTION
**Pull request by Makwe-O**
#### What does this PR do
Refactor the response model
#### Description of the task to be completed?
Currently, the response model includes 3 columns (rate, check and text area) which contain the different types of questions. The PR refactors this implementation by removing the 3 columns and adding a new column(question_type) which contains the different types of question in an Enum format and adds a response column to contains the response to the question.

#### What are the relevant pivotal tracker stories?
[#165854187](https://www.pivotaltracker.com/story/show/165854187)

#### Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have linted my code prior to submission
- [x] My changes generate no new warnings
- [x] Existing unit tests pass locally with my changes
- [x] Implementation works according to expectations

### CODE REVIEW

File: helpers/response/create_response.py [hunk 1]
```diff
@@ -1,51 +1,61 @@
...
-            suggestion.save()
-            responses.append(suggestion)
+    elif question_type.lower() == 'input' and \
+            'text_area' in kwargs and kwargs['text_area']:
+        suggestion = Response(
+            response=kwargs['text_area'],
+            room_id=kwargs['room_id'],
+            question_id=kwargs['question_id'],
+            created_date=datetime.now())
+        suggestion.question_type = 'textarea'
```
Comment (KMaina) [inline helpers/response/create_response.py]: Good job @Makwe-O, kindly shed some light on something I have noticed. On line 49 we are querying for question type of `input` but on line 56 we are saving it as `textarea` in response. Just for uniformity, could we save both as `input`?

**Comment by Makwe-O** [comment]
> Please look into your commit messages, use engineering playbook to guide you

Sorry I missed that. I will get it fixed

**Comment by Bonifase** [comment]
> Please look into your commit messages, use engineering playbook to guide you

@Makwe-O

File: api/question/schema_query.py [hunk 2]
```diff
@@ -54,7 +54,7 @@ def get_room_response(self, unique_responses, question_id, question_type):
             responses_in_room = ResponseModel.query.filter_by(room_id=room.room_id, question_id=question_id)  # noqa: E501
             response_count = ResponseModel.query.filter_by(room_id=room.room_id).count()  # noqa: E501
             if question_type == "rate":
-                average_rating = sum([response.rate for response in responses_in_room]) / responses_in_room.count()  # noqa: E501
+                average_rating = sum([response.response for response in responses_in_room]) / responses_in_room.count()  # noqa: E501
```
Comment (daniellamarr) [inline api/question/schema_query.py]: Do you think you can refactor the lines that have the **noqa: E501**

**Pull request merged**